pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -20,10 +20,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var BundleKind = "Bundle"
+var (
```

Comment:
```suggestion
const (
```

If changing this, why not use constants instead? I don't understand why these values are declared as variables. Seems like constants to me, but there might be a reason.

Reply:
I was following as it was done by someone else - var declaration. Yes, constant variables are better in this particular case.